script/poll-for-es

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+#
+# This script will poll the Elasticsearch health endpoint until the cluster
+# reaches a yellow state which is good enough for testing. This script will poll
+# for up to 30 seconds waiting for Elasticserach to start. It will give up at
+# that time and return a non-zero exit code.
+
+es_port=${ES_PORT:-9200}
+count=0
+
+until $(curl -s "localhost:${es_port}/_cluster/health?wait_for_status=yellow&timeout=30s" > /dev/null 2>&1); do
```

Comment:
This is handy. I might switch to using this on Travis CI. I used `wget` with its built-in retries but it's a little opaque how long it will try for.